### DISCUSSION TITLE
Is there a way to toggle off the gyro?

### DISCUSSION THREAD

**Discussion by Yetil829** [AntiMicroX/antimicrox#737] [Q&A]
Is there a way to toggle off the gyro on a controller? Using 8bitdo Pro 2.

I was also wondering if there was a way to use the Gyro of an 8bitdo Pro 2 while in Xiniput mode.

**Comment by maxigaz** [comment]
You could try creating an additional mapping set where you don't have anything mapped to the gyro, and [toggle between the two sets](https://github.com/AntiMicroX/antimicrox/wiki/Set-Selection) with a press of a button.

**Comment by pktiuk** [comment]
There is a linked feature request: https://github.com/AntiMicroX/antimicrox/issues/892
